DirectProgramming/DPC++FPGA/Tutorials/GettingStarted/fpga_compile/README.md

```diff
@@ -109,7 +109,7 @@ The compiler options used are explained in the table.
 | `-fintelfpga`      | Perform ahead-of-time compilation for FPGA.
 | `-DFPGA_EMULATOR`  | Adds a preprocessor define that invokes the emulator device selector in this sample (see code snippet above).
 | `-Xshardware`      | `-Xs` is used to pass arguments to the FPGA backend. <br> Since the emulator is the default FPGA target, you must pass `Xshardware` to instruct the compiler to target FPGA hardware.
-| `-Xsboard`         | Optional argument to specify the FPGA board target. <br> If omitted, a default FPGA board is chosen.
+| `-Xstarget`         | Optional argument to specify the FPGA board target. <br> If omitted, a default FPGA board is chosen.
```

Comment:
Might want to update this comment.

Reply:
what do you think this should say? @bsyrowik 

This? 

> Optional argument to specify the FPGA target device. If omitted, a default FPGA board is chosen.

Reply:
Yeah, that seems fine!